### ISSUE TITLE
Windows 10: after upgrading to latest version - I get an error (was working before updating)

### ISSUE THREAD

**Issue by cintia-baniola** [angular/angular-cli#1242]
> ng serve
module.js:327
    throw err;
    ^

Error: Cannot find module 'rsvp'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\Hans\AppData\Roaming\npm\node_modules\angular-cli\node_modules\leek\lib\leek.js:3:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)

Hans@HP-HANS C:\Users\Hans\Web\frontend\platform\js\angular20\angular2-questions-manekinekko
> ng version
module.js:327
    throw err;
    ^

Error: Cannot find module 'rsvp'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\Hans\AppData\Roaming\npm\node_modules\angular-cli\node_modules\leek\lib\leek.js:3:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)

Hans@HP-HANS C:\Users\Hans\Web\frontend\platform\js\angular20\angular2-questions-manekinekko
>

**Comment by neilhem** [comment]
Try to remove node_modules folder & reinstall it again

**Comment by popstr** [comment]
Removing node_modules folder and running npm install did not solve the issue for me. I removed angular-cli and installed it again and then it worked.
npm remove -g angular-cli
npm install -g angular-cli

**Comment by indcoder** [comment]
Even I had the same issue a few days back. After numerous npm install failed for angular-cli , the .npm modules got fried.

I would follow the steps @popstr has mentioned but perhaps add an intermediate command between npm remove and install

`npm cache clean`

But if that doesnt work....then we have to go for the nuke option.
So this is what I did , I upgraded node from v5.* to v6.2.2(the latest). I had shift-deleted my .npm and .npm cache folder in the Roaming directory in Windows.
